deployment/v3/utils/ssl_renewel_util/renew_ssl_certs.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Script to renew ssl certificates..
+
+# Prompt the user for the expiry date
+read -p "Enter the expiry date of existing ssl certs (e.g., 2024-12-31): " expiry_date
+
+# Define the source and destination directories
+src_dir="/etc/letsencrypt/live"
+dest_dir="/etc/letsencrypt/live-${expiry_date}"
+
+# Check if the source directory exists
+if [ -d "$src_dir" ]; then
+    # Move the directory
+    sudo mv "$src_dir" "$dest_dir"
+    echo "Directory moved to $dest_dir"
+else
+    echo "Source directory $src_dir does not exist."
+fi
+
+# Regenerate the ssl certificates
+# Prompt the user for the domain name
+read -p "Enter the domain name (e.g., sandbox.xyz.net): " domain
```

Comment:
if the domain env variable exist if not prompt for domain

Reply:
use `mosip_domain` as env variable

Reply:
Updated